### DISCUSSION TITLE
Seeking ideas for new features

### DISCUSSION THREAD

**Discussion by gwlucastrig** [gwlucastrig/Tinfour#111] [General]
I am seeking ideas for new features to add to the Tinfour software library.  

I planning on returning to work on Tinfour starting in February of 2025.  In preparation for that effort, I am asking my users for suggestions on features or capabilities that would make the software more useful to them. 

For example, Tinfor has a limitation that prevents an application from adding new vertices to the Delaunay triangulation after constraints are specified (if constraints are not specified, an application can add or remove vertices at will).   I would like to overcome that limitation.

While I cannot promise that I can do everything that users might want, I will give all suggestions due consideration.

Finally, as always, I thank my users for their support and hope that Tinfour continues to prove useful to their applications.

Gary

**Comment by micycle1** [comment]
Refinement!

**Reply by gwlucastrig** [reply]
Agreed!

**Comment by micycle1** [comment]
Perhaps something of interest here: https://www.inf.usi.ch/hormann/papers/Gotsman.2021.OLD.pdf

**Reply by gwlucastrig** [reply]
Thanks.  A lot to digest here.  I'm going to go through it and see what kind of Tinfour API extensions are needed to support an algorithm of this type.  It looks like a fun and challenging project.
